.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: node_js
-node_js:
-  - "lts/*"
-  - "node"
+
+matrix:
+  include:
+    - os: linux
+      node_js: "node"
+    - os: windows
+      node_js: "node"
+
 after_success:
```

Comment:
you need to add test in` autodeploy.sh` to deploy only on linux os.

Reply:
Well seen